src/katex.less

```diff
@@ -32,13 +36,6 @@
     // mode, while still allowing background/foreground setting on root .katex
     * { -ms-high-contrast-adjust: none !important; }
 
-    .katex-html {
-        // Making .katex inline-block allows line breaks before and after,
-        // which is undesireable ("to $x$,"). Instead, adjust the .katex-html
```

Comment:
Are you able to verify that in `to $x$,` it's impossible to have a break between the `x` and comma? In a narrow container it should break before `x` instead.

Reply:
No, in fact this is #1233. I've left this comment as is, though it clearly is incorrect, and I can edit it. This PR has no effect on this issue.

Reply:
Huh. I swear this used to work.